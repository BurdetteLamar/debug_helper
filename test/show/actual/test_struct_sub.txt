---
DebugHelperTest::MyStructSub (message='test_struct_sub'):
  DebugHelperTest::MyStructSub#size: 3
  Member 0:
    Name:
      Symbol:
        Symbol#to_s: foo
        Symbol#size: 3
        Symbol#encoding: !ruby/encoding US-ASCII
    Value: Integer 0
  Member 1:
    Name:
      Symbol:
        Symbol#to_s: bar
        Symbol#size: 3
        Symbol#encoding: !ruby/encoding US-ASCII
    Value: Integer 1
  Member 2:
    Name:
      Symbol:
        Symbol#to_s: baz
        Symbol#size: 3
        Symbol#encoding: !ruby/encoding US-ASCII
    Value: Integer 2
