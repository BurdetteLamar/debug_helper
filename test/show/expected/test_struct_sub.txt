---
DebugHelperTest::MyStructSub (size=3 message='test_struct_sub'):
  size: 3
  Member 0:
    Name:
      Symbol:
        to_s: foo
        size: 3
        encoding: !ruby/encoding US-ASCII
    Value: Fixnum 0
  Member 1:
    Name:
      Symbol:
        to_s: bar
        size: 3
        encoding: !ruby/encoding US-ASCII
    Value: Fixnum 1
  Member 2:
    Name:
      Symbol:
        to_s: baz
        size: 3
        encoding: !ruby/encoding US-ASCII
    Value: Fixnum 2
